packages/emails/package.json

```diff
@@ -10,20 +10,19 @@
   "main": "./src/index.ts",
   "types": "./src/index.ts",
   "dependencies": {
-    "@aws-sdk/client-ses": "^3.292.0",
-    "@aws-sdk/credential-provider-node": "^3.292.0",
-    "@react-email/components": "0.0.2",
-    "@react-email/render": "0.0.6",
-    "clsx": "^1.2.1",
-    "nodemailer": "^6.9.1",
+    "@aws-sdk/client-ses": "^3.501.0",
+    "@aws-sdk/credential-provider-node": "^3.501.0",
+    "@react-email/components": "^0.0.14",
+    "@react-email/render": "^0.0.12",
+    "nodemailer": "^6.9.8",
     "preview-email": "^3.0.19",
-    "react-email": "^1.9.1"
+    "react-email": "^2.0.0"
```

Comment:
Is this dependency really needed? There are no imports from `react-email`, only `@react-email/*. `react-email` has **a lot** of dependencies and some of them are clearly dev dependencies.